containers/Dockerfile.centos7-foss2020a

```diff
@@ -0,0 +1,17 @@
+FROM easybuilders/test:centos7-eb421-gcc930
+USER easybuild
+# use /easybuild for everything that should persist in the container (software, modules and sources)
+# everything else goes in /scratch, which will be cleaned out
+ENV EASYBUILD_PREFIX=/scratch
```

Comment:
as we discussed in our VOLUME quiz. You might just add /scratch as a volume in the base-image, no need to clean that up anymore.
On the flip-side - you might confuse people if they inherit from an image and scratch is gone. Rather discuraged to use VOLUME for that reason. But possible nontheless